src/data/resolvers/user.ts

```diff
@@ -41,6 +44,35 @@ const getUserReputation = async (
   return reputationAmount;
 };
 
+const getUserLock = async (
+  client: ApolloClient<object>,
+  colonyManager: ColonyManager,
+  walletAddress: Address,
+  tokenAddress: Address,
+): Promise<UserLock> => {
+  const { networkClient } = colonyManager;
+  const tokenLockingClient = await networkClient.getTokenLockingClient();
```

Comment:
This is not the correct way of getting the token locking client. 

I'm actually surprised this worked.

Apply this patch:
```diff
diff --git a/src/lib/ColonyManager/ColonyManager.ts b/src/lib/ColonyManager/ColonyManager.ts
index 0b41f7682..20b1e995a 100644
--- a/src/lib/ColonyManager/ColonyManager.ts
+++ b/src/lib/ColonyManager/ColonyManager.ts
@@ -9,8 +9,10 @@ import {
   Extension,
   OneTxPaymentClient,
   TokenClient,
+  TokenLockingClient,
   ExtensionClient,
 } from '@colony/colony-js';
+import getTokenLockingClient from '@colony/colony-js/lib/clients/TokenLockingClient';
 
 import ENS from '~lib/ENS';
 import { isAddress } from '~utils/web3';
@@ -27,6 +29,8 @@ export default class ColonyManager {
 
   tokenClients: Map<Address, Promise<TokenClient>>;
 
+  tokenLockingClients: Map<Address, TokenLockingClient>;
+
   networkClient: ColonyNetworkClient;
 
   provider: Provider;
@@ -37,6 +41,7 @@ export default class ColonyManager {
     this.colonyClients = new Map();
     this.extensionClients = new Map();
     this.tokenClients = new Map();
+    this.tokenLockingClients = new Map();
     this.networkClient = networkClient;
     this.provider = networkClient.provider;
     this.signer = networkClient.signer;
@@ -117,6 +122,11 @@ export default class ColonyManager {
     identifier?: AddressOrENSName,
   ): Promise<TokenClient>;
 
+  async getClient(
+    type: ClientType.TokenLockingClient,
+    identifier?: AddressOrENSName,
+  ): Promise<TokenLockingClient>;
+
   async getClient(
     type: ClientType.OneTxPaymentClient,
     identifier?: AddressOrENSName,
@@ -146,6 +156,18 @@ export default class ColonyManager {
         const colonyClient = await this.getColonyClient(identifier);
         return colonyClient.tokenClient;
       }
+      case ClientType.TokenLockingClient: {
+        if (!identifier)
+          throw new Error('Need colony identifier to get TokenLockingClient');
+        const colonyClient = await this.getColonyClient(identifier);
+        const tokenLockingClient = this.getTokenLockingClient(identifier);
+        /*
+         * @TODO This needs a proper fix in colonyJS
+         */
+        // @ts-ignore
+        colonyClient.tokenLockingClient = tokenLockingClient;
+        return tokenLockingClient;
+      }
       case ClientType.OneTxPaymentClient: {
         if (!identifier)
           throw new Error('Need colony identifier to get OneTxPaymentClient');
@@ -173,4 +195,13 @@ export default class ColonyManager {
     }
     return clientPromise;
   }
+
+  async getTokenLockingClient(address: Address): Promise<TokenLockingClient> {
+    let clientPromise = this.tokenLockingClients.get(address);
+    if (!clientPromise) {
+      clientPromise = getTokenLockingClient(address, this.signer);
+      this.tokenLockingClients.set(address, clientPromise);
+    }
+    return clientPromise;
+  }
 }
```

Then get it from the colonyManager like this:

```
await colonyManager.getClient(
  ClientType.TokenLockingClient,
  colonyAddress,
);